### ISSUE TITLE
Training Results

### ISSUE THREAD

**Issue by bcba25** [RizhaoCai/Learn-Convolutional-Neural-Network-for-Face-Anti-Spoofing#2]
Hi, could you please provide you training results, whats the validation accuracy and which training set you used ? Thanks
